stories/Introduction.stories.mdx

```diff
@@ -0,0 +1,211 @@
+import { Meta } from '@storybook/addon-docs';
```

Comment:
@IgorChaikin Could you please add only components, which are used in project, for example: `Item`, `Order`, `Categories`, etc.

<img width="1428" alt="image" src="https://user-images.githubusercontent.com/12561302/128711960-90d93997-dc51-442d-b96c-75e3aefd5115.png">